apps/api/controllers/players/show.rb

```diff
@@ -0,0 +1,21 @@
+module Api
+  module Controllers
+    module Players
+      class Show
+        include Api::Action
+
+        expose :player
+
+        def initialize(interactor: Interactors::Players::FetchPlayer.new)
+          @interactor = interactor
+        end
+
+        def call(params)
+          player = @interactor.call(params[:id]).player
+          halt 400 && self.status = 400 if player.nil?
```

Comment:
We probably want to use `404` here (not found).

Reply:
Do we need to explicitly set `status` when using `halt`?

Reply:
@jgayfer fixed.